.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: java
 jdk:
   - openjdk8
+env:
+  global:
+    - GRAALVM_VERSION=19.0.2
+    - GRAALVM_HOME=/opt/graalvm-ce-$GRAALVM_VERSION
 cache:
   directories:
     - $HOME/.m2/repository
     - $HOME/.m2/wrapper
+    - $GRAALVM_HOME
+before_script:
+  - if [ ! -f "$GRAALVM_HOME/bin/java" ]; then
+      curl https://github.com/oracle/graal/releases/download/vm-${GRAALVM_VERSION}/graalvm-ce-linux-amd64-${GRAALVM_VERSION}.tar.gz | tar xvz -C /opt;
```

Comment:
that's fine but we can eventually use the docker build provided by the quarkus maven plugin

Reply:
> that's fine but we can eventually use the docker build provided by the quarkus maven plugin

What would be the advantage of that?

Reply:
To always use the right version of Substrate VM for the version of quarkus we target

Reply:
That sounds good and I'd be all for using it, but I am not finding any single piece of docs how to set it up. Do you happen to have an example at hand?

Reply:
You can find some info here: https://quarkus.io/guides/building-native-image-guide.html